docs/install_guides/installing-openvino-linux-header.md

```diff
@@ -6,21 +6,13 @@
    :maxdepth: 3
    :hidden:
 
-   Using Installer <openvino_docs_install_guides_installing_openvino_linux>
```

Comment:
Instead of installer, we need to add a way to use archives. They provide the same content as Installer, but in a different form. 

CC @artanokhov